src/pages/PeoplePage/PeoplePage.tsx

```diff
@@ -0,0 +1,50 @@
+import { useEffect, useState } from 'react';
+import { getPeople } from '../../api';
+import { Loader } from '../../components/Loader';
+import { PeopleTable } from '../../components/PeopleTable';
+import { Person } from '../../types';
+
+export const PeoplePage = () => {
+  const [people, setPeople] = useState<Person[]>([]);
+  const [errorMessage, setErrorMessage] = useState('');
+  const [loading, setLoading] = useState(false);
+
+  useEffect(() => {
+    setLoading(true);
+    setErrorMessage('');
+
+    setTimeout(() => {
```

Comment:
The `setTimeout` function is used here without a delay parameter, which defaults to 0. If the intention is not to simulate a delay, consider removing `setTimeout` to simplify the code.